selfdrive/car/docs_definitions.py

```diff
@@ -31,60 +30,68 @@ class Star(Enum):
   EMPTY = "empty"
 
 
-class Harness(Enum):
-  nidec = "Honda Nidec"
-  bosch_a = "Honda Bosch A"
-  bosch_b = "Honda Bosch B"
-  toyota = "Toyota"
-  subaru_a = "Subaru A"
-  subaru_b = "Subaru B"
-  fca = "FCA"
-  ram = "Ram"
-  vw = "VW"
-  j533 = "J533"
-  hyundai_a = "Hyundai A"
-  hyundai_b = "Hyundai B"
-  hyundai_c = "Hyundai C"
-  hyundai_d = "Hyundai D"
-  hyundai_e = "Hyundai E"
-  hyundai_f = "Hyundai F"
-  hyundai_g = "Hyundai G"
-  hyundai_h = "Hyundai H"
-  hyundai_i = "Hyundai I"
-  hyundai_j = "Hyundai J"
-  hyundai_k = "Hyundai K"
-  hyundai_l = "Hyundai L"
-  hyundai_m = "Hyundai M"
-  hyundai_n = "Hyundai N"
-  hyundai_o = "Hyundai O"
-  hyundai_p = "Hyundai P"
-  hyundai_q = "Hyundai Q"
-  custom = "Developer"
-  obd_ii = "OBD-II"
-  gm = "GM"
-  nissan_a = "Nissan A"
-  nissan_b = "Nissan B"
-  mazda = "Mazda"
-  ford_q3 = "Ford Q3"
-  ford_q4 = "Ford Q4"
-  none = "None"
-
-
-class HarnessPart(Enum):
+class CarPart(Enum):
```

Comment:
can you separate the part categories with comments? like `# harnesses`, `# cables`, `# mounts` etc